subsys/logging/log_core.c

```diff
@@ -203,6 +219,18 @@ int log_init(void)
 	return 0;
 }
 
+void log_thread_set(k_tid_t process_tid)
```

Comment:
Should we allow to replace logger internal thread?

Reply:
It is exposed in case user does not use internal thread. I can add assert there to check if proc_tid before setting is null.